server/comms/msgjson/types.go

```diff
@@ -106,36 +107,7 @@ const (
 	NotifyFeeRoute = "notifyfee"
 )
 
-// Bytes is a byte slice that marshals to and unmarshals from a hexadecimal
-// string. The default go behavior is to marshal []byte to a base-64 string.
-type Bytes []byte
-
-// BytesFromHex is a Bytes constructor that accepts a hex string.
-func BytesFromHex(s string) (Bytes, error) {
-	b, err := hex.DecodeString(s)
-	return Bytes(b), err
-}
```

Comment:
I removed this function because it is interchangeable with `hex.DecodeString` where is was being used. I saw that @dnldd was using this in #49 though, so I'm happy to add it back if it's needed.